ros_installation.md

```diff
@@ -0,0 +1,197 @@
+
+## Using the default Gazebo version for a ROS distribution using binary installations
+
+For the users that need to run a specific version of ROS and want to use all
+the gazebo ROS related packages out-of-the-box, this is the recommended
+section:
+
+### ROS2 Humble and ROS2 Rolling
+ROS2 Humble and ROS2 Rolling support officially the Gazebo Fortress version.
+For a fully-integrated ROS system, Fortress is the recommened version. The way
+to proceed is just to use the ROS repository and do ***not*** use the
+osrfoundation repository.
+
+ROS packages `ros-humble-ros-gz` and `ros-rolling-ros-gz` will be available
+pulling the necessary Gazebo Fortress dependencies.
+
+### ROS2 Galactic
+ROS2 Galactic supports officially the Gazebo Edifice
+version. For a fully-integrated ROS system, Edifice is the recommened
+version. The way to proceed is just to use the ROS repository and do ***not***
+use the osrfoundation repository.
+
+ROS packages `ros-galactic-ros-gz` will be available pulling the necessary
+Gazebo Edifice dependencies.
+
+### ROS2 Foxy
+ROS2 Foxy supports officially the Gazebo Citadel
+version. For a fully-integrated ROS system, Citadel is the recommened
+version. The way to proceed is just to use the ROS repository and do ***not***
+use the osrfoundation repository.
+
+ROS packages `ros-foxy-ros-gz` will be available pulling the necessary
+Gazebo Citadel dependencies.
+
+### ROS1 Noetic
+ROS1 Noetic supports officially the Gazebo Citadel
+version. For a fully-integrated ROS system, Citadel is the recommened
+version. The way to proceed is just to use the ROS repository and do ***not***
+use the osrfoundation repository.
+
+ROS packages `ros-noetic-ros-gz` will be available pulling the necessary
+Gazebo Citadel dependencies.
+
+## Using an specific Gazebo version with ROS2
+***Warning!:*** Using this option, you won't be able to use any ROS Ubuntu package
+related to Gz libraries from ROS deb repository.
+
+Selecting a different release of Gz than the one officially supported in each
+ROS distribution will require to compile the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) from source. Same applies for
+other ROS packages using Gz libraries that could also require code changes in
+they are not supporting the new Gz version chosen.
+
+### Gz Garden
+
+Gz Garden can be used with ROS2 Humble and ROS2 Rolling, the [`ros_gz
+bridge`](https://github.com/gazebosim/ros_gz) supports the compilations from
+source.
+
+ * Folow [the instruction to install gz-garden](https://gazebosim.org/docs/garden/install_ubuntu#binary-installation-on-ubuntu)
+   from osrfoundation.org repository
+ * Follow the instructions to compile the ros_gz bridge from source in a colcon workspace
+   * [ROS2 Humble](https://github.com/gazebosim/ros_gz/tree/humble#from-source)
+     * Be sure of using `export GZ_VERSION=humble`
+   * [ROS2 Rolling](https://github.com/gazebosim/ros_gz/tree/ros2#from-source)
+     * Be sure of using `export GZ_VERSION=rolling`
```

Comment:
This is wrong, maybe `ROS_VERSION`?

Reply:
ouch, [4dd64b9](https://github.com/gazebosim/docs/pull/339/commits/4dd64b91e508fd7fc8e856d6a719ad2e65ed3b98) . Good catch.